projects/vdk-core/plugins/vdk-plugin-control-cli/src/vdk/plugin/control_cli_plugin/execution_skip.py

```diff
@@ -0,0 +1,164 @@
+        This method calls the "/data-jobs/for-team/{team_name}/name/{job_name}/executions"
+        endpoint with the additional params "submitted" and "running". The API will return all jobs that are
+        submitted and running with the provided team and job name. We have the following possible scenarios:
+        API returns an empty list - this means no other data jobs excecutions are running and we can proceed with current execution.
+        API returns one execution with a different ID - this means that another excecution is running and we should skip.
+        API returns one job with the same ID - this means that we can proceed with excecution.
+        API returns two jobs one of which has a different ID - we need to skip current excecution.
+        It then parses the response and returns a boolean.
+        :return: bool
+                 If the data job has a different execution running in K8S.
+                 A running different execution means that there is a data job execution currently
+                 submitted or running with a different id and in that case we should skip current execution
+                 until the other finishes.
+        """
+        jobs = self._get_running_job_executions(job_name, job_team)
+        job_execution_running = False
+        for job_execution in jobs:
+            # We break only if we have a job with a different execution id
+            # Sometimes ids returned by VDK execution_state can slightly differ than
+            # ids returned by the API. VDK ids  have '-xxxxx' appended at the end,
+            # where x is a random alphameric char. Otherwise ids start the same.
+            if not job_execution_attempt_id.startswith(job_execution.id):
+                log.info(
+                    f"Found another running job execution for {job_name}. Current execution id: {job_execution_attempt_id}. Other running execution id: {job_execution.id}"
+                )
+                job_execution_running = True
+                break
+
+        return job_execution_running
+
+    def _get_running_job_executions(
+        self, job_name, team_name
+    ) -> List[DataJobExecution]:
+        job_status = [
+            "submitted",
+            "running",
+        ]  # We query only for running or manual jobs
+        response = self.execution_api_client.data_job_execution_list(
+            job_name=job_name, team_name=team_name, execution_status=job_status
+        )
+        log.info(f"Api call returned job execution list: {response}")
+        return response
+
+    @staticmethod
+    def _authenticate(api_token_authorization_url, api_token, auth_type) -> None:
+        log.info("Authenticating.")
+        auth = Authentication()
+        auth.update_api_token_authorization_url(api_token_authorization_url)
+        auth.update_api_token(api_token)
+        auth.update_auth_type(auth_type)
+        auth.acquire_and_cache_access_token()
+        log.info("Login successful.")
```

Comment:
Same as previous comment; either make these log calls more descriptive or get rid of them.